activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -137,6 +137,32 @@ def test_exec_insert_default_values_quoted_schema_with_returning_disabled_and_no
         assert_equal expect.to_i, result.rows.first.first
       end
 
+      def test_enable_extension
+        connection = connection_without_insert_returning
+        connection.create_schema "other_schema"
+
+        connection.enable_extension(:hstore)
+        assert connection.extension_enabled?("hstore")
+
+        connection.disable_extension("hstore")
+        assert_not connection.extension_enabled?("hstore")
+
+        connection.enable_extension("hstore")
+        assert connection.extension_enabled?("hstore")
+
+        connection.disable_extension("hstore")
+        assert_not connection.extension_enabled?("hstore")
+
+        connection.enable_extension("other_schema.hstore")
+        assert connection.extension_enabled?("hstore")
+
+        connection.disable_extension("hstore")
+        assert_not connection.extension_enabled?("hstore")
+
+        ensure
+          connection.drop_schema "other_schema", if_exists: true
```

Comment:
It looks like 'end' is missing here.

Reply:
The `end` is on the following line :)

Reply:
I should have clearly mentioned that this warning needs addressed.

https://buildkite.com/rails/rails/builds/88742#018284e3-fe31-4750-a12a-e82159ee53cd/1183-1188
```ruby
/rails/activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb:162: warning: mismatched indentations at 'ensure' with 'def' at 140
```